ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -338,6 +338,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
+- as: aws-ipi-rm-root-secret-f14
```

Comment:
This job is in a 4.12->4.13 upgrade jobs, but from the doc, seem like the case is only applicable for z-stream upgrade case?

Reply:
The [note](https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/cco-mode-mint.html#manually-removing-cloud-creds_cco-mode-mint
) from doc:

> Prior to a non z-stream upgrade, you must reinstate the credential secret with the administrator-level credential. If the credential is not present, the upgrade might be blocked.

Removing y upgrade for this scenario. cc @shellyyang1989